src/main/java/org/terasology/launcher/game/Installation.java

```diff
@@ -110,15 +142,11 @@ static Semver getEngineVersion(Path versionDirectory) throws IOException {
      * @return true iff the file matches the expected pattern for Terasology's engine JAR
      */
     static boolean matchEngineJar(Path path, BasicFileAttributes basicFileAttributes) {
-        // Are there path-matching utilities to simplify this?
```

Comment:
the answer is probably something like [Apache Commons IO](https://commons.apache.org/proper/commons-io/) [WildcardFileFilter](https://commons.apache.org/proper/commons-io/apidocs/org/apache/commons/io/filefilter/WildcardFileFilter.html)

Reply:
(that's just a note, certainly not required for this PR)

Reply:
that's your own question btw :yum:

Reply:
yes I know ;)